### PR TITLE
Update Rescue in ContentProvidersController

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Add `rescue` for when `fetch` throws `ActiveRecord::NoRecordFound`
* Add `rescue` to both `show` and `edit` methods
* Update specs

[#120786217]

### CODE REVIEW

File: app/controllers/think_feel_do_engine/bit_maker/content_providers_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ class ContentProvidersController < ApplicationController
       before_action :authenticate_user!, :set_arm,
                     :set_content_modules, :set_slideshows
 
-      PROVIDER_NOT_FOUND = "The slide you are looking for no longer exists."
+      PROVIDER_NOT_FOUND = "Unable to find ContentProvider"
```
Comment (eschlange) [inline app/controllers/think_feel_do_engine/bit_maker/content_providers_controller.rb]: I'm not sure this message would mean anything to a participant. They don't understand what a ContentProvider is.

File: app/controllers/think_feel_do_engine/bit_maker/content_providers_controller.rb [hunk 2]
```diff
@@ -92,6 +89,9 @@ def destroy
 
       def find_content_provider
         ContentProviderDecorator.fetch(params[:id])
+      rescue ActiveRecord::RecordNotFound
+        redirect_to arm_bit_maker_content_providers_url(@arm),
```
Comment (eschlange) [inline app/controllers/think_feel_do_engine/bit_maker/content_providers_controller.rb]: Nice improvement here.

**Comment by eschlange** [comment]
@michaelwehrley Other than my one concern, everything looks good. Nice additional spec coverage.

File: app/controllers/think_feel_do_engine/bit_maker/content_providers_controller.rb (hunk 1 above)
Comment (michaelwehrley) [inline app/controllers/think_feel_do_engine/bit_maker/content_providers_controller.rb]: True.  This is available to a user.

**Pull request merged**